### ISSUE TITLE
为什么ovearleaf 无法编译成功？

### ISSUE THREAD

**Issue by wukongwong** [hithesis/hithesis#153]
## 描述bug类型

- 校区（哈尔滨/威海/深圳？）：

- 学位（本/硕/博？）：

- 论文类型（毕业论文/开题/中期/博后/English Version？）：

- LaTeX 发行版（TeXLive 20xx/MikTeX/**本模板不支持CTeX套装**）：

- 模板版本（在 `hithesis***.cls` 中的 `\ProvidesClass` 命令中的 `3.x.xx` 即为版本号）：

## 描述bug具体信息

- 请给出完整的示例文件：

```latex
% 将 thesis.tex/report.tex 中的内容完整地放在这里

```

或将整个 `hitbook/chinese`, `reports` 或 `reportplus` 压缩后上传 `.zip` 文件

- 使用什么编译方式（`xelatex`/`latexmk`/`make thesis`）：
- 编译报错 or 编译结果与预期不符：

- 如果有编译报错，可以将报错信息截图粘贴到下面：

- 如果编译结果与预期不符：
  - 预期结果的截图

  - 编译后的效果截图

- 将编译后产生的 `.log` 文件上传到这里。

**Comment by syvshc** [comment]
将 `.latexmkrc` 文件中的 

```
preview_continuous_mode
```

的值改为 `0`

**Comment by wukongwong** [comment]
Yes, it works. I suggest making it clear in the readme. Thanks a lot!
